bitcoin/src/amount.rs

```diff
@@ -2328,7 +2301,7 @@ mod tests {
     #[test]
     fn disallow_unknown_denomination() {
         // Non-exhaustive list of unknown forms.
-        let unknown = vec!["NBTC", "UBTC", "ABC", "abc"];
+        let unknown = vec!["NBTC", "UBTC", "ABC", "abc", "cBtC"];
```

Comment:
```suggestion
        let unknown = vec!["NBTC", "UBTC", "ABC", "abc", "cBtC", "Sat", "Sats", "Msat"];
```
Only if you agree with my arguments above of course :)

Reply:
done :+1: